packages/cli/config/jest.js

```diff
@@ -43,7 +43,6 @@ const transformIgnorePattern = [
   'highlight\\.js',
   'prismjs',
   'json-schema',
-  'react-use',
```

Comment:
Hm does this mean that if someone out there DOES still use `/lib/`, they are screwed? Remember, this CLI is shipped and used worldwide. Or what's the impact of this line?

Reply:
This needs a changeset too BTW. And if we do keep the change, the changeset should probably very loudly say that users of `react-use` should switch the way that they do their imports.

Reply:
Rebased.
I have also added changeset to cli, and tried to minimize disruption in tests, by leaving `react-use/lib` in the list.
This way both `react-use/lib` and `react-use/esm` work in tests.